NAMESPACE

```diff
@@ -3,11 +3,15 @@
 S3method(fit_model,RtGam_bam)
 S3method(fit_model,RtGam_gam)
 S3method(fit_model,default)
+S3method(predict,RtGam)
 S3method(print,RtGam)
 export(RtGam)
 export(check_diagnostics)
 export(dataset_creator)
 export(dates_to_timesteps)
 export(penalty_dim_heuristic)
+export(predict_obs_incidence)
```

Comment:
namming does this need to be obs_incidence vs obs?

Reply:
I'm trying to differentiate between observed cases & incidence and the to-be-written functions to get fitted cases & incidence.

Reply:
i.e. posterior predicted vs. expectation

Reply:
I see - I guess I am not clear what the difference is between incidence and cases but I need to look more at the code. I think I would be tempted to say leave that to predict to make clear (which suggests in the future that the Rt element of this should move to the fitted functionality in the future)

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/48/commits/1836493540a3f726c06ce6b82cf0d1322d6ffa48